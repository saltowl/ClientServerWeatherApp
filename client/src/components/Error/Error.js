import React from 'react';
import './Error.scss';

class Error extends React.Component {
  constructor(props) {
    super(props);
  }
  
  render() {
    return (
      <div className="Error">
          <div>Cannot {this.props.message}</div>
      </div>
    );
  }
}

export default Error;
