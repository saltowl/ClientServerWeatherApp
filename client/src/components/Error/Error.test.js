import React from "react";
import Error from "./Error";
import renderer from "react-test-renderer";

it("Error display correctly", () => {
  const message = "delete city";
  const component = renderer.create(<Error message={message} />);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
