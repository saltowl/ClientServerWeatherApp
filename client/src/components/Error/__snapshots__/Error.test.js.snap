// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error display correctly 1`] = `
<div
  className="Error"
>
  <div>
    Cannot 
    delete city
  </div>
</div>
`;
